app/src/main/java/com/github/se/orator/ui/overview/SpeakingPracticeModule.kt

```diff
@@ -67,17 +66,19 @@ fun SpeakingPracticeModule(
                         imageVector = Icons.AutoMirrored.Filled.ArrowBack,
                         contentDescription = "Back",
                         modifier = Modifier.size(dimensions.iconSizeSmall),
-                        tint = AppColors.textColor)
+                        tint = MaterialTheme.colorScheme.onSurface)
                   }
             },
             colors =
                 TopAppBarDefaults.centerAlignedTopAppBarColors(
-                    containerColor = AppColors.surfaceColor,
-                    titleContentColor = AppColors.textColor))
+                    containerColor = MaterialTheme.colorScheme.surfaceContainer,
+                    titleContentColor = MaterialTheme.colorScheme.onSurface))
       },
       content = { paddingValues ->
         Column(modifier = Modifier.fillMaxSize().padding(paddingValues)) {
-          Divider()
+          HorizontalDivider(
```

Comment:
Maybe also here, using a colorScheme color to make it easier for dark mode transition. 
We could try to modify the colorscheme set we already have to bring it closer to our needs.

Reply:
Hey ! sorry but I don't get where do you suggest the change ? I think line 80 contains the `MaterialTheme.colorScheme` you are referring to.